### PR TITLE
Adding instructions for installing and running the bot in the repository.

### PR DESCRIPTION
**Pull request by riyasingh1004**
## Explanation
I have added some instructions in the readme about installing and running oppiabot in a repository. This might be helpful to new contributors and also added command for running the tests locally.

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by riyasingh1004** [comment]
@vojtechjelinek @jameesjohn Can you kindly review this PR.

File: README.md [hunk 1]
```diff
@@ -71,7 +71,8 @@ Please refer to the following instructions to setup Oppiabot for the first time
 
 ## Installing the bot on a repository
 
-You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/apps/your-app-name`
+You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/apps/your-app-name`. In the `.env` file put your github account name in 
+`WHITELISTED_ACCOUNTS` and also add your repository (name in small caps) in the `constants.js` file locally.
```
Comment (jameesjohn) [inline README.md:75]: > also add your repository (name in small caps) in the `constants.js` file locally

Why is this required?

**Review by jameesjohn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (riyasingh1004) [reply README.md:75]: I think we need to add the name of our repository along with the events in constants.js file for running oppiabot in the repository, as I did that too.

**Review by jameesjohn — APPROVED** [review]
LGTM
